index.js

```diff
@@ -141,20 +141,38 @@ NYC.prototype.addAllFiles = function () {
   this.writeCoverageFile()
 }
 
+var hashCache = {}
```

Comment:
I suppose it doesn't matter in practice but may be better to have this on the instance?